lib/pages/home/upcoming_events_card.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:acs_upb_mobile/pages/timetable/model/events/uni_event.dart';
+import 'package:acs_upb_mobile/pages/timetable/service/uni_event_provider.dart';
+import 'package:acs_upb_mobile/widgets/info_card.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:acs_upb_mobile/generated/l10n.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:time_machine/time_machine.dart';
+
+extension EventExtension on UniEventInstance {
+  String dateStringRelativeToToday(BuildContext context) {
```

Comment:
Oh and I just remembered - in an update to the internationalization package, they actually removed the need to have the context to get a localized string, see [this issue](https://github.com/localizely/flutter-intl-intellij/issues/4). You should be able to call `S.current.labelToday`. Therefore this could potentially be just a getter instead of a method.

Reply:
Solved